eventstore/tasks.py

```diff
@@ -349,3 +349,72 @@ def mark_turn_contact_healthcheck_complete(msisdn):
         },
     )
     response.raise_for_status()
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded),
+    retry_backoff=True,
+    max_retries=15,
+    acks_late=True,
+    soft_time_limit=10,
+    time_limit=15,
+)
+def archive_turn_conversation(urn, message_id, reason):
+    headers = {
+        "Authorization": "Bearer {}".format(settings.TURN_TOKEN),
+        "Accept": "application/vnd.v1+json",
+        "Content-Type": "application/json",
+    }
+
+    data = json.dumps({"before": message_id, "reason": reason})
+
+    r = requests.post(
+        urljoin(settings.TURN_URL, f"v1/chats/{urn}/archive"),
+        headers=headers,
+        data=data,
+    )
+    r.raise_for_status()
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded, TembaHttpError),
+    retry_backoff=True,
+    max_retries=15,
+    acks_late=True,
+    soft_time_limit=10,
+    time_limit=15,
```

Comment:
10 seconds is probably way too low for going through all the waiting for helpdesk contacts